api-docs/cloud-dedicated/ref.yml

```diff
@@ -1813,9 +1832,44 @@ paths:
       summary: Query using the InfluxDB v1 API
       tags:
         - Query
+        - Data I/O endpoints
   /write:
     post:
       operationId: PostLegacyWrite
+      description: |
+        Writes data to a database.
+
+        Use this endpoint to send data in [line protocol](/influxdb/cloud-dedicated/reference/syntax/line-protocol/) format to InfluxDB.
+
+        InfluxDB Cloud Dedicated does the following when you send a write request:
+
+        1.  Validates the request.
+        2. If successful, attempts to [ingest the data](/influxdb/cloud-dedicated/reference/internals/durability/#data-ingest); [error](/influxdb/cloud-dedicated/write-data/troubleshoot/#review-http-status-codes) otherwise.
+        3.  Processes data in the request body and then responds with one of the following HTTP status codes:
+          - `204 Success`: All points in the batch have been written and are queryable.
+          - `400 Bad request`: Some points were rejected. The response body contains details about the [rejected points](/influxdb/cloud-dedicated/write-data/troubleshoot/#troubleshoot-rejected-points). Points that were written are queryable.
```

Comment:
201 when partial-writes are on, and some lines were written.
Otherwise, still a 400.